src/main/java/org/jolokia/docker/maven/access/chunked/ChunkedResponseHandler.java

```diff
@@ -1,5 +1,7 @@
 package org.jolokia.docker.maven.access.chunked;
 
+import org.jolokia.docker.maven.access.DockerAccessException;
+
 public interface ChunkedResponseHandler<T> {
-    void process(T toProcess);
+    void process(T toProcess) throws DockerAccessException;
```

Comment:
Updating the interface contract to allow for exception throwing. `DockerAccessException` may not be the best choice based on its description, but everything seemed to funnel into it anyway, so I went with that instead of creating a whole new class of exception.